.jshintrc

```diff
@@ -1,7 +0,0 @@
-{
-  "undef": true,
-  "unused": true,
-  "node": true,
-  "esversion": 6,
-  "eqeqeq": true
-}
```

Comment:
jshint might left the cool quadrant of js universe but it is still better to have it inside until conversion to eslint hasn't been done ❓